zomes/blog/capabilities/main/main.rs

```diff
@@ -0,0 +1,32 @@
+extern crate hdk;
+extern crate serde_json;
+use std::time::SystemTime;
+
+fn create_post(input: serde_json::Value) -> serde_json::Value {
```

Comment:
I started off with both input and output being ```String```s and having a serde deserialize command as the first line in each function, but since we have to wrap these functions in a ```_dispatch()``` through our HDK anyway, why not use ```serde_json::Value``` here?

Reply:
Talking with @lucksus it becomes clear that the deeper issue here is that this also is about cross-language parameter handling (i.e. that works for any language calling in and out of the wasm), i.e. that inputs and outputs are always JSON. This makes sense to me because it means things are handled natively for typscript, and every other wams language will certainly have JSON handling.  Change request is that we document this in an ADR.

Reply:
The json!() thing is nice...